src/Core/Framework/Adapter/Twig/Extension/PcreExtension.php

```diff
@@ -35,18 +36,18 @@ public function pregReplace(string $subject, string $pattern, string $replacemen
         $value = preg_replace($pattern, $replacement, $subject);
 
         if ($value === null) {
-            throw new \RuntimeException(\sprintf('Error while processing Twig preg_replace filter. Error: %s', preg_last_error_msg()));
+            throw AdapterException::couldNotProcessRegexp(preg_last_error_msg());
         }
 
         return $value;
     }
 
     public function pregMatch(string $subject, string $pattern): bool
     {
-        $result = preg_match($pattern, $subject);
+        $result = @preg_match($pattern, $subject);
```

Comment:
same here

Reply:
here it is handled with `$result === false` check afterwards right?

Reply:
yes